python/docs/source/reference/pyspark.sql.rst

```diff
@@ -482,9 +484,9 @@ Functions
     sequence
     sha1
     sha2
-    shiftLeft
-    shiftRight
-    shiftRightUnsigned
+    shiftleft
+    shiftright
+    shiftrightunsigned
```

Comment:
I should rename it here - the case sensitivity is not taken into account `sphinx.ext.autosummary` due to the file name conflicts at RST stubs.